addon/templates/data-table.hbs

```diff
@@ -7,6 +7,7 @@
             {{yield (hash
             column=(component 'dt-column-header')
             selectionColumn=(component (if singleSelection 'dt-column-header' 'dt-selection-column-header') isSelected=isAllSelected selected=(action 'selectAll') deselected=(action 'deselectAll'))
+            sortableColumn=(component 'dt-sortable-column-header')
```

Comment:
the problem with this approach, that I can have different components for different columns (quick example, different comparison functions)

How about consider external `columns` array with specific column options like:
```
 columns: [
    Ember.Object({key: 'someLabel', header: 'Some Label', sorting: 'asc'}),
    Ember.Object({key: 'place', header: 'Place'}),
    Ember.Object({key: 'zone', header: 'Zone'}),
    Ember.Object({key: 'date', header: 'Date', sorting: true, component: 'my-table-cell-date'})
  ],
```

I will create example in upcoming days